### PR TITLE
Add buildspec for AWS CodePipeline

### PR DESCRIPTION
**Pull request by nvengal**
This file defines the environment and commands that will be used to deploy this project in CodePipeline/CodeBuild.

### CODE REVIEW

File: app/controllers/api/base_controller.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@ module Api
   class BaseController < ApplicationController
     rescue_from ActiveRecord::RecordNotFound, with: :missing_record
 
+    private
```
Comment (nvengal) [inline app/controllers/api/base_controller.rb:8]: This change is unrelated to this pr, but I noticed it while working on the deployment. All public methods in controllers get turned into lambdas by Jets and there's no need for us to have a `missing_record` lambda function.

File: buildspec.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    commands:
+      # Jets doesn't yet support ruby 2.7
+      # Let's temporarily install 2.5 until we can upgrade
+      #   https://github.com/tongueroo/jets/issues/444
+      - rbenv install 2.5.8
+      - gem install bundler
+  build:
+    commands:
+      - echo "Starting ${BUILD_PHASE} deploy using $(ruby --version)"
+      - bundle install
```
Comment (maheshkurma) [inline buildspec.yml:19]: Question: Is there any reason for not having migrations command here.
`bundle exec jets db:migrate`

Comment (nvengal) [reply buildspec.yml:19]: The pantry-finder database is read only. We don't want to execute migrations from the application. Currently, this api exposed a datasource that is modified and loaded externally.

Comment (maheshkurma) [reply buildspec.yml:19]: got it.

Comment (davidpickering) [reply buildspec.yml:19]: @maheshkurma thanks for thinking it through to that extent. For reference, here is our general PT-FT architecture:

image

**Pull request merged**